docs/tutorials/videos/markdown-syntax.md

```diff
@@ -0,0 +1,19 @@
+# Markdown Syntax
+
+With Taipy GUI, you can easily create a basic application page using Markdown syntax. This covers things like connecting variables and adding visual elements like sliders and charts.
+
+Markdown syntax is a simple way to format plain text and structure documents. It lets you easily add things like headings, lists, links, images, and emphasis 
+to text without needing complex HTML or formatting tools.
+
+Taipy uses [Python Markdown](https://python-markdown.github.io/) to turn your Markdown text into web pages. There are several language extensions that help make your pages look nice and user-friendly. 
+Specifically, Taipy uses the following Markdown extensions: 
+
+- [Admonition](https://python-markdown.github.io/extensions/admonition/) 
```

Comment:
Not sure if this us so interesting but we can keep it

Reply:
Agreed Florian. Every useless detail should be removed. To the point.